docs/server-elasticsearch-setup.md

```diff
@@ -7,31 +7,28 @@ sidebar_label: Set up Elasticsearch
 ## Overview
 
 You can integrate [Elasticsearch](https://www.elastic.co/elasticsearch/) with the Temporal Server to enhance [Workflow search](/docs/server-workflow-search) functionality.
-Temporal Server versions < 1.6.0 require the use of Kafka + Zookeeper to use Elasticsearch.
-Temporal Server versions >= 1.6.0 do not require any additional dependencies to use Elasticsearch.
-The [Server versions and dependencies](/docs/server-versions-and-dependencies) page lists compatible versions of dependencies.
+The [Server versions and dependencies](/docs/server-versions-and-dependencies/#workflow-search) page describes which versions require the use of Kafka, and which ones do not.
 
 Integration with Elasticsearch is defined in the Server configuration files.
-For Server versions >= 1.6.0, all you need to do is edit the `persistence` section.
-For Server versions < 1.6.0, you will need to edit the `kafka` section as well.
+If you are not using Kafka, all you need to do is edit the `persistence` section.
+If you are using Kafka, you will need to edit the `kafka` section as well.
 
 You can use the Temporal Server [development_es.yaml](https://github.com/temporalio/temporal/blob/master/config/development_es.yaml) file as working example.
 
 ## Persistence
 
 Elasticsearch relies on storage to operate.
 Storage related configurations are defined within the `persistence` section of the configuration file.
-The Temporal Server needs to know where to connect to the Elasticsearch storage.
```

Comment:
The last sentence is ok. I don't like the first one. We are not describing how ES works here, right? I would replace these 3 sentences with something like: "Elasticsearch is a search engine. The Temporal Server needs to know where to connect to the Elasticsearch. Add the `advancedVisibilityStore: es-visibility` key value pair to the `persistence` section."

Reply:
No, but we are trying to make the connection between configuring persistence (storage) and Elasticsearch.
I.e why am editing a section called "persistence" when I am trying to set up a search engine?